internal/database/etcd/etcd.go

```diff
@@ -91,35 +100,40 @@ func (etcd *Etcd) readByte() byte {
 // Read reads data from etcd and returns p bytes to user
 func (etcd *Etcd) Read(p []byte) (n int, err error) {
 	if etcd.ID == uuid.Nil {
-		err = errors.New("please create a new etcd client using NewWithID")
-		return n, err
+		return 0, errors.New("please create a new etcd client using NewWithID")
 	}
 
 	key := fmt.Sprintf("%s/%s", etcd.treePrefix, etcd.ID.String())
 
 	if !etcd.hasRead {
 		resp, err := etcd.client.Get(etcd.ctx, key)
 		if err != nil {
-			return n, err
+			return 0, err
```

Comment:
This change is unnecessary since "n" is a named return value with default being 0

Reply:
Making it obvious for readability.

Reply:
I disagree, but don't care enough :)